build.gradle

```diff
@@ -40,6 +39,14 @@ version = hasProperty("module_version") ? "$project.module_version" : 'unknown'
 repositories {
     mavenLocal()
     jcenter()
+    maven {
+        name = "GitHubPackages"
+        url = uri("https://maven.pkg.github.com/aiven/aiven-kafka-connect-commons")
+        credentials {
+            username = System.getenv("GITHUB_ACTOR")
+            password = System.getenv("GITHUB_TOKEN")
```

Comment:
Are these credentials needed: the package repo is public? It makes it difficult to build locally.

Reply:
It used only for publishing the package to the GitHub Packages. It is public, but you need GitHub credentials to publish a package.

Reply:
Gotcha